jetty-start/src/main/java/org/eclipse/jetty/start/fileinits/MavenLocalRepoFileInitializer.java

```diff
@@ -219,6 +290,65 @@ public Coordinates getCoordinates(URI uri)
         return coords;
     }
 
+    protected void download(Coordinates coords, Path destination)
+        throws IOException
+    {
+        if (coords.version.endsWith("-SNAPSHOT"))
```

Comment:
I don't think this is necessary at all - seems to introduce a lot of additional complexity.
Maybe it's a generic feature but I don't think it should go into this PR.

Reply:
As discussed, this is ok.